src/rofl/datapath/pipeline/openflow/openflow1x/pipeline/of1x_flow_table.h

```diff
@@ -172,6 +172,13 @@ rofl_result_t __of1x_destroy_table(of1x_flow_table_t* table);
 */
 
 /**
+* @internal Add flow entry to table being able to match cookie
+*
+* @warning Out of spec.
```

Comment:
Actually the docs right here. You just added the warning, but in case somebody wants to use this API call it should be documented. Otherwise somebody might even remove it, since its 'out of spec' and the use and consequences of this function might not be obvious to everyone.